sortinghat/core/db.py

```diff
@@ -470,6 +544,10 @@ def add_enrollment(uidentity, organization,
         when `start < MIN_PERIOD_DATE`; or `end > MAX_PERIOD_DATE`
         or `start > end`.
     """
+    input_args = {'uidentity': uidentity.uuid,
+                  'organization': organization.name,
+                  'start': copy.deepcopy(str(start)), 'end': copy.deepcopy(str(end))}
+
```

Comment:
+1 to the formatting and to the renaming!

About keeping the arguments closer to the call, most of the times it is not possible because they are modified in the method, that is why they are saved in the beginning of the method. i have added a comment in these cases to make this clearer.